django_loci/base/admin.py

```diff
@@ -246,6 +248,17 @@ def __init__(self, *args, **kwargs):
             self.fields['floorplan'].choices = floorplan_choices + [
                 (floorplan.pk, floorplan)
             ]
+        # setting attributes basis user permissions
+        if (
+            user
+            and user.has_perm('django_loci.view_objectlocation')
+            and not user.has_perm('django_loci.change_objectlocation')
```

Comment:
I am not sure this will work in openwisp-controller.

Reply:
@nemesifier pushed a commit which hopefully should solve all the above issues. Please review at your convenience! Thank you